worker-jobtracking-container/src/main/startup.d/setup-database.sh

```diff
@@ -16,8 +16,9 @@
 #
 
 
-cd /database
-java -jar job-service-db-installer.jar \
+cd /maven
+java -cp "*:classpath" \
```

Comment:
I know it's harmless but why do you keep adding `:classpath` to all these?

```suggestion
java -cp "*" \
```